alert.css

```diff
@@ -0,0 +1,83 @@
+:root {
+    --primary-color: #006D77;
+    --error: rgb(241, 55, 55);
+    --warning: rgb(247, 184, 67);
+    --success: green;
+    --white: #fff;
+    --black:#000;
+}
+
+body {
+    font-family: Roboto,sans-serif;
+}
+
+.alert_container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+}
+
+.alert {
+    border-radius: 0.3rem;
+    padding: 0.5em;
+    font-size: 1rem;
+    width: 30%;
+    box-sizing: border-box;
+}
+
+.alert p {
+    margin: 0.5rem;
+}
+
+.alert_info {
+    border: 1px solid;
+}
+
+/* Alert with background color */
+
+.alert_primary, 
+.alert_warn,
+.alert_error,
+.alert_success {
+    color: var(--white);
+}
+
+.alert_primary {
+    background-color: #006D77;
+}
+
+.alert_warn {
+    background-color: orange;
+}
+
+.alert_error {
+    background-color: red;
+}
+
+.alert_success {
+    background-color: green;
+}
+
+/* Alert with outlined border */
+
+.alert_primary_outlined {
+    border: 1px solid var(--primary-color);
+    color: var(--primary-color);
+}
+
+.alert_warn_outlined {
+    border: 1px solid var(--warning);
+    color: var(--warning);
+}
+
+.alert_error_outlined {
+    border: 1px solid var(--error);
```

Comment:
suggestion you can use  color name instead of naming error,warning. Also as above in background color u can define those colors to root and use var instead. rest look good to me.

Reply:
Done.